Reword sentence in auto-generated docstrings.

mkcor opened the pull request:
This is a tiny PR inspired from my reviewing https://github.com/plotly/dash-bio/pull/411. Why the distance (expressed by "those")? I believe the meaning we want is that of "these."

## Contributor Checklist

- [ ] I have broken down my PR scope into the following TODO tasks
   -  [ ] task 1
   -  [ ] task 2
- [ ] I have run the tests locally and they passed. (refer to testing section in [contributing](https://github.com/plotly/dash/blob/master/CONTRIBUTING.md))
- [ ] I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR

### optionals

- [ ] I have added entry in the `CHANGELOG.md`
- [ ] If this PR needs a follow-up in **dash docs**, **community thread**, I have mentioned the relevant URLS as follow
    -  [ ] this github [#PR number]() updates the dash docs
    -  [ ] here is the show and tell thread in plotly dash community

alexcjohnson commented:
hmm, to me it actually sounds a little better with "Those" - but I'd actually prefer us to rewrite it just using (potentially nested) indentation, rather than duplicating the keys (so those/these will disappear entirely). Also we need to rewrap lines so the indentation will be preserved in auto-wrapping contexts. To take a simple example, here's the current `active_cell` from the table:
```
- active_cell (dict; optional): The row and column indices and IDs of the currently active cell. active_cell has the following type: dict containing keys 'row', 'column', 'row_id', 'column_id'.
Those keys have the following types:
  - row (number; optional)
  - column (number; optional)
  - row_id (string | number; optional)
  - column_id (string; optional)
```

This should be cleaned up to look something like:
```
- active_cell (dict; optional): The row and column indices and IDs of the
  currently active cell.
  active_cell is a dict with keys:
  - row (number; optional)
  - column (number; optional)
  - row_id (string | number; optional)
  - column_id (string; optional)
```

mkcor commented:
It should be "These" because we're referring to those keys we *just* mentioned... ;)
Anyhow, I like the deeper change you're suggesting.

I also agree on the formatting improvement. Speaking of which, I've been wondering whether "active_cell is a dict with keys:" should be "`active_cell` is a dict with keys:" (the way other docstrings are in the SciPy ecosystem). Maybe this has been discussed and decided before.

Leaving that ;) detail aside, I'll rework the PR.

mkcor commented:
@shammamah I've taken care of Dash Bio components in terms of running `black` over there: https://github.com/plotly/dash-bio/pull/420

mkcor commented:
I have rebased (getting rid of the original "Those" --> "These" change) and rewritten the new docstrings we want in the tests (TDD, so tests are expected to fail at 97dd9e71f7983b152117c89c8317c0834b06514e).

There are four instances of "Those keys have the following types:" left (two in `test_metadata_conversions.py` and two in `test_flow_metadata_conversions.py`) because I don't see where the previous sentence "... dict containing keys ..." begins. If I don't figure it out, I'll leave it to another PR.

mkcor commented:
Ok, I'm afraid this is getting unreasonable... This is the kind of diff I'm battling with when testing docstrings:

![ouch_diff](https://user-images.githubusercontent.com/2227806/65506022-eba91e80-deca-11e9-851f-6f4b3bab5a88.png)

One culprit I believe is:

https://github.com/plotly/dash/blob/75cbcd8ee133904cc4531391cee8cced0188cc5b/dash/development/_py_components_generation.py#L487

This value should not be constant but instead inherit, to account for the nested cases.

I'll take care of the general reformatting issue (including `black` in the Python component generation here in Dash, replacing https://github.com/plotly/dash-bio/pull/420) in a new PR, which will be small and beautiful.

mkcor commented on dash/development/_py_components_generation.py:
```diff
@@ -449,8 +449,8 @@ def create_prop_docstring(prop_name, type_object, required, description,
-            '{description}{period}' \
-            '{name} has the following type: {type}'.format(
+            '{description}{period}\n' \
+            '{indent_spacing}{name} is a {type}'.format(
```
This should be:
```suggestion
            '{indent_spacing}  {name} is a {type}'.format(
```

alexcjohnson commented:
@mkcor I'm afraid we never got around to finishing this and it's gone stale... hopefully if/when we finish #1429 we can take these ideas into account.
